src/plugins/data/common/expressions/interpreter_provider.ts

```diff
@@ -17,19 +17,25 @@
  * under the License.
  */
 
-import clone from 'lodash.clone';
-import { each, keys, last, mapValues, reduce, zipObject } from 'lodash';
-import { getType, fromExpression, getByAlias, castProvider } from '@kbn/interpreter/common';
+/* eslint-disable @typescript-eslint/no-var-requires */
+import { clone, each, keys, last, mapValues, reduce, zipObject } from 'lodash';
+
+// @ts-ignore
+import { fromExpression, getType, getByAlias, castProvider } from '@kbn/interpreter/target/common';
```

Comment:
We should be able to import from `@kbn/interpreter/common` directly, I’ll look into why that doesn’t work.